### PR TITLE
fix the aws region that hardcoded to "us-east-1" 

### PR DESCRIPTION
**Pull request by yancl**
When i run `kops` in some isolated region, it does not work as expected.

the error message looks like this:
```
Using config file: /home/vagrant/.kops.yaml
I1121 09:05:25.618724   21083 s3context.go:82] Querying S3 for bucket location for "xxx"

error reading cluster configuration "xxx": error reading s3://xxx/config: error getting location for S3 bucket "xxx": InvalidAccessKeyId: The AWS Access Key Id you provided does not exist in our records.
	status code: 403, request id: xxx
```

So i add an env `KOPS_AWS_REGION` which can be overriden in such situation:)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/961)

### CODE REVIEW

File: upup/pkg/fi/cloudup/awsup/aws_utils.go [hunk 1]
```diff
@@ -38,7 +38,11 @@ func ValidateRegion(region string) error {
 		glog.V(2).Infof("Querying EC2 for all valid regions")
 
 		request := &ec2.DescribeRegionsInput{}
-		config := aws.NewConfig().WithRegion("us-east-1")
+		awsRegion := os.Getenv("KOPS_AWS_REGION")
```
Comment (kris-nova) [inline upup/pkg/fi/cloudup/awsup/aws_utils.go]: Is this a new ENV VAR that we will need to be supporting?

Is this documented anywhere?

Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/awsup/aws_utils.go]: Should we default to `us-east-1` and allow for overriding?

Comment (yancl) [inline upup/pkg/fi/cloudup/awsup/aws_utils.go]: Yes, allow for overriding will be great!

Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/awsup/aws_utils.go]: Please allow overriding based on AWS vars.

**Comment by chrislovecnm** [comment]
@k8s-bot test this

**Comment by yancl** [comment]
@chrislovecnm sorry, response too late. I have changed the name according AWS env var spec, please have a look, thanks:)

**Comment by kris-nova** [comment]
This LGTM - Although we probably need to add a note on this feature when we look at this issue https://github.com/kubernetes/kops/issues/1124

**Pull request merged**